.github/ISSUE_TEMPLATE/rse-group.yml

```
@@ -1,5 +1,5 @@
 name: Research Software Group
-description: Add your group to our list
+description: Fill out this issue form to list your group
 title: "[Group]: "
```

Thanks Daan for my basket change. I do think this title is a bit unclear. How about 

```suggestion
title: "[Your group name]: "
```